lib/trento_web/views/v1/sap_system_view.ex

```diff
@@ -127,6 +127,20 @@ defmodule TrentoWeb.V1.SapSystemView do
 
   def render("database_deregistered.json", %{id: id, sid: sid}), do: %{id: id, sid: sid}
 
+  def render("database_instance_deregistered.json", %{
```

Comment:
I'm just finding this views redundant (together with some added like those). They don't do anything, and they are not even tested.
I don't know how worthy is to have them...
Anyway, just thinking aloud